hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/fs/ozone/TestRootedOzoneFileSystem.java

```diff
@@ -1513,4 +1529,65 @@ public void testRenameDestinationParentDoesntExist() throws Exception {
     }
   }
 
+  @Test
+  public void testNonPrivilegedUserMkdirCreateBucket() throws IOException {
+    // This test is only meaningful when ACL is enabled
+    if (!enableAcl) {
```

Comment:
I think we should use org.junit.Assume.assumeTrue(enableAcl) here to avoid the test run in cases where acl-s are disabled. I think it is a more standard solution to avoid test runs in case of a specific parameter value.

Reply:
Done